R/bootstrap_model.R

```diff
@@ -7,15 +7,25 @@
 #' @param ... Arguments passed to or from other methods.
 #' @inheritParams p_value
 #'
-#' @return A data frame.
+#' @return A data frame of bootstrapped estimates.
+#'
+#' @section Using with \code{emmeans}:
+#' The output can be passed directly to the various functions from the
+#' \code{emmeans} package, to obtain bootstrapped estimates, contrasts, siple
+#' slopes, etc, and their confidence intervals.
 #'
 #' @seealso \code{\link{bootstrap_parameters}}, \code{\link{simulate_model}}, \code{\link{simulate_parameters}}
 #'
 #' @examples
 #' \donttest{
 #' if (require("boot")) {
-#'   model <- lm(mpg ~ wt + cyl, data = mtcars)
-#'   head(bootstrap_model(model))
+#'   model <- lm(mpg ~ wt + factor(cyl), data = mtcars)
+#'   b <- bootstrap_model(model)
+#'   print(head(b))
+#'
+#'   if (require("emmeans")) {
+#'     print(emmeans(b, consec ~ cyl))
```

Comment:
So why is it `consec` here?

Reply:
No particular reason - just wanted to show off some other contrasts methods other than `pairwise`...

Reply:
Ah, ok. I thought that was a variable name...